_data-prepper/pipelines/contains.md

```diff
@@ -27,11 +27,89 @@ This call returns `true` if the field `message` contains the substring `abcd` or
 Alternatively, you can use a literal string as the first argument:
 
 ```
-contains('This is a test message', 'test')
+'contains("This is a test message", "test")'
 ```
 {% include copy.html %}
 
 In this case, the function returns `true` because the substring `test` is present within the string `This is a test message`.
 
 The `contains()` function performs a case-sensitive search.
 {: .note}
+
+## Example
+
+The following pipeline uses `contains()` to add a boolean flag `has_test` based on a substring in `/message` and to filter out non-matching events, forwarding only messages containing "ERROR" to OpenSearch:
+
+```yaml
+contains-demo-pipeline:
+  source:
+    http:
+      ssl: false
+
+  processor:
+    - add_entries:
+        entries:
+          - key: "has_test"
+            value_expression: 'contains(/message, "test")'
```

Comment:
Thanks for using single quotes here!

Reply:
@dlvenable its actually not necessary, I've removed them now